Sources/DreamsViewController.swift

```diff
@@ -25,18 +25,22 @@ public class DreamsView: UIView {
     internal var webService: WebServiceType = WebService()
     internal var dreams = Dreams.shared
 
-    public override init(frame: CGRect) {
-        super.init(frame: frame)
+    public override init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: Bundle?) {
+        super.init(nibName: nil, bundle: nil)
```

Comment:
Do you think we could pass the `nibName` and `bundle` here?

Reply:
Yes for sure! 🙇